crates/sui-core/src/transaction_manager.rs

```diff
@@ -93,7 +95,134 @@ impl LockQueue {
     }
 }
 
-#[derive(Default)]
+struct CacheInner {
+    versioned_cache: LruCache<ObjectID, SequenceNumber>,
+    // we cache packages separately, because they are more expensive to look up in the db, so we
```

Comment:
non-system package? I think system packages will ne in versioned cache?

Reply:
system packages (framework and move stdlib) aren't referred to directly by transactions so they don't show up here anyway